qcore/geo.py

```diff
@@ -7,6 +7,7 @@
 from warnings import warn
 
 import numpy as np
+from numba import njit
```

Comment:
Need to add numba to the requirements file as it's a new requirement for the repo